src/c++/perf_analyzer/model_parser.h

```diff
@@ -151,6 +153,23 @@ class ModelParser {
   /// model
   bool ResponseCacheEnabled() const { return response_cache_enabled_; }
 
+  /// Returns whether or not top level request caching is enabled for this model
+  /// \return the truth value of whether top level request caching is enabled
+  /// for this model
+
+  bool TopLevelRequestCachingEnabled() const
```

Comment:
Can you use `Response` caching instead of `Request` caching everywhere for consistency?

Reply:
I will fix this.